input/fsh/SD_Bundle.fsh

```diff
@@ -2,7 +2,27 @@ Profile: MCODEPatientBundle
 Parent: Bundle
 Id: mcode-patient-bundle
 Title: "mCODE Patient Bundle"
-Description: "A collection of data for an mCODE cancer patient."
+Description: "A collection of data for an mCODE cancer patient. The bundle MUST contain exactly one Patient resource conforming to the CancerPatient profile, and one or more Condition resource(s) conforming to the PrimaryCancerCondition profile. The bundle MUST also contain following mCODE-conformant resources, if available to the server:
+
+* Condition resources conforming to SecondaryCancerCondition
+* Observation resources conforming to CancerDiseaseStatus
+* Observation resources conforming to PerformanceStatus (ECOG and/or Karnofsky)
+* Observation resources conforming to TumorMarker test results
+* MedicationRequest resources conforming to CancerRelatedMedicationRequests (prescriptions, administrations, and patient-reported medications)
+* Procedure resources conforming to CancerRelatedProcedure (surgical and radiological)
+* Observation resources conforming to CancerStage (Clinical Group and TNM and/or Pathological Group and TNM)
+* Observation resources conforming to CancerGeneticVariant
+* DiagnosticReport resources conforming to CancerGenomicsReport
+* Specimen resources conforming to GeneticSpecimen
+* Observation resources conforming to GenomicRegionStudied
+* Observation resource conforming to CancerRelatedComorbidities, and accompanying Condition resources 
+* Observation resources conforming to VitalSigns (height, weight, blood pressure)
```

Comment:
Since `VitalSigns` is not an mCODE profile, I think it makes sense to link to the documentation.

```suggestion
* Observation resources conforming to [VitalSigns](http://hl7.org/fhir/R4/observation-vitalsigns.html) (height, weight, blood pressure)
```